server/config/config_test.go

```diff
@@ -225,4 +225,16 @@ secret_struct_slice_flag:
 		require.NoError(t, err)
 		require.Equal(t, []secretHolder{{Secret: "FIRST\nSECRET"}, {Secret: "SECOND\nSECRET"}}, *secretStructSliceFlag)
 	}
+
+	// Parse errors are propagated
```

Comment:
Can you move this testcase to a standalone function? Although the bug was introduced as part of the "expansion" feature, the bug itself has nothing to do with it.

Reply:
Done - moved to `TestLoadFromData`, with both a parse-working and parse-broken test. Does this naming look ok, tests follow the conventions you all prefer?